strong-config/node
# feat: add CLI with encrypt and decrypt commands

Donezo:

- [x] Add `oclif`-based CLI with `encrypt`, `decrypt` and `validate` commands including required arguments and flags
- [ ] Support all major key providers for encryption.
- [x] Tests. These are finicky: Works when running isolatedly (per-file), but fail when running with `yarn test`. My next attempt is to go a more jest-native approach without `@oclif/test` and with spies.
    - [x] Tests for `decrypt`
    - [x] Tests for `validate`
    - [x] Tests for `encrypt`

How to test:
- Install strong config locally or globally. This should make the CLI available (`strong-config` or `yarn strong-config`)
- Run `strong-config decrypt --help`, `strong-config encrypt --help` and `strong-config validate --help` and explore the CLI. Examples and usage are part of the `--help` output. You can pass the example configs and schemas under `example/`
